docs/gc.md

```diff
@@ -83,17 +76,17 @@ The *nursery* area is subdivided into two further areas, the *allocate* space an
 
-Within the *tenure* area, new objects are allocated into the small object area (SOA), which is illustrated in the earlier diagram (see 3.). A large object area (LOA) is set aside for objects greater than 64 KB that cannot be allocated into the SOA to minimize fragmentation. The LOA is allocated by default but is reduced and removed after a few GC cycles if it isn't populated. To prevent the creation of an LOA, you can specify the [`-Xnoloa`](xloa.md) option on the command line when you start your application. When the *tenure* area is close to full a global GC is triggered.
+Within the *tenure* area, new objects are allocated into the small object area (SOA), which is illustrated in the earlier diagram (see item 3). A large object area (LOA) is set aside for objects greater than 64 KB that cannot be allocated into the SOA to minimize fragmentation. The LOA is allocated by default but is reduced and removed after a few GC cycles if it isn't populated. To prevent the creation of an LOA, you can specify the [`-Xnoloa`](xloa.md) option on the command line when you start your application. When the *tenure* area is close to full a global GC cycle is triggered.
 
-The local GC scavenge reduces pause times by freqently reclaiming memory in the *nursery* area which, for a transactional application with many short-lived objects, has the most recyclable space. However, over time the *tenure* area might become full. So, whilst a local GC scavenge process is operating on the *nursery* area, a concurrent global GC process also runs alongside normal program execution to mark and remove unreachable objects from the *tenure* area. These two GC approaches combine to provide a good trade-off between shorter pause times and consistent throughput.
+The partial GC cycle (*scavenge*) reduces pause times by frequently reclaiming memory in the *nursery* area which, for a transactional application with many short-lived objects, has the most recyclable space. However, over time the *tenure* area might become full. So, whilst a partial GC cycle is operating on the *nursery* area, a concurrent global GC cycle also runs alongside normal program execution to mark and remove unreachable objects from the *tenure* area. These two GC approaches combine to provide a good trade-off between shorter pause times and consistent throughput.
```

Comment:
There is a missing sentence to connect the flow.

... has the most recyclable space. While most of the objects will stay in nursery after scavenge is complete, a small fraction (above tenure age) will be moved to tenure.  Over time the *tenure* area might become full....

Reply:
Changed:

"...has the most recyclable space. While most of the objects stay in the *nursery* area after the *scavenge* operation is complete, a small fraction are moved to the *tenure* area. Over time...."

Previous para explains tenure age, so I don't think we need to spell this out here.